svthalia/concrexit
# Profile page crash

Closes #1352 

### Summary
Conditional check if member parameter in there in the request field in ProfileDetailView 

### How to test
Steps to test the changes y:
1. Open https://thalia.nu/members/profile/ in Incognito mode
2. See it redirects you to login page
